filters/scheduler/lifo_test.go

```diff
@@ -301,6 +303,38 @@ func TestNewLIFO(t *testing.T) {
 	}
 }
 
+type testTracer struct {
+	*mocktracer.MockTracer
+	spans int32
+}
+
+func (t *testTracer) Reset() {
+	atomic.StoreInt32(&t.spans, 0)
+	t.MockTracer.Reset()
+}
+
+func (t *testTracer) StartSpan(operationName string, opts ...opentracing.StartSpanOption) opentracing.Span {
+	atomic.AddInt32(&t.spans, 1)
+	return t.MockTracer.StartSpan(operationName, opts...)
+}
+
+func (t *testTracer) FinishedSpans() []*mocktracer.MockSpan {
+	timeout := time.After(1 * time.Second)
+	retry := time.NewTicker(100 * time.Millisecond)
+	defer retry.Stop()
+	for {
+		finished := t.MockTracer.FinishedSpans()
```

Comment:
https://github.com/opentracing/opentracing-go/pull/239 adds `UnfinishedSpans()` but the library is deprecated so I it seems there will be no new release